staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -284,3 +286,64 @@ type ObjectReference struct {
 	// +optional
 	Subresource string `json:"subresource,omitempty" protobuf:"bytes,7,opt,name=subresource"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AuditConfiguration represents a cluster level audit configuration
+type AuditConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	// Policy to enforce on the given backend
+	// if undefined it will default to the statically configured cluster policy if available
+	// if neither exist the backend will fail
+	// +optional
+	Policy *Policy `json:"policy,omitempty" protobuf:"bytes,2,opt,name=policy"`
```

Comment:
> if undefined it will default to the statically configured cluster policy if available

What will happen if the policy is defined but has zero rules set?
What will happen if the policy is defined and OmitStages is set but has zero rules set?

Reply:
in these cases the policy validation will catch and provide useful feedback